### PR TITLE
Introduce support for Gutenberg

### PR DESCRIPTION
**Pull request by EvanHerman**
Resolves issue https://github.com/godaddy/wp-primer-theme/issues/239

At the moment, the cover photo block within Gutenberg does not apply the `.alignfull` & `.alignwide` classes to the block container. This is to be addressed in a future release, so the `.alignfull` class will have an effect on the cover photo block.

Gutenberg Image Block Reference:
https://github.com/WordPress/gutenberg/issues/2650

### CODE REVIEW

File: functions.php [hunk 1]
```diff
@@ -127,6 +127,17 @@
 
 }
 
+/**
+ * Load Beaver Builder compatibility file.
```
Comment (frankiejarrett) [inline functions.php]: @EvanHerman Fix copypasta

File: functions.php [hunk 2]
```diff
@@ -127,6 +127,17 @@
...
+ *
+ * @since 1.0.0
```
Comment (frankiejarrett) [inline functions.php]: @EvanHerman `@since NEXT`

File: inc/compat/gutenberg.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Gutenber compatibility.
```
Comment (frankiejarrett) [inline inc/compat/gutenberg.php]: @EvanHerman No g love

File: inc/compat/gutenberg.php [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ *
+ * @package    Compatibility
+ * @subpackage Gutenberg
+ * @category   Core
+ * @author     GoDaddy
+ * @since      1.0.0
```
Comment (frankiejarrett) [inline inc/compat/gutenberg.php]: @EvanHerman `@since NEXT`

File: style.css [hunk 5]
```diff
@@ -1,3 +1,4 @@
+@charset "UTF-8";
```
Comment (frankiejarrett) [inline style.css:1]: @EvanHerman This looks like a regression of a bug we fixed a while back in https://github.com/godaddy/wp-primer-theme/commit/f0da8e4a3021218926675b6e3280439c8970c634. Looks like we need to understand the root cause and fix it.

Comment (EvanHerman) [reply style.css:1]: @fjarrett Looks like `replace:charset` runs on default `grunt` task, but not on `grunt:watch`

**Comment by frankiejarrett** [comment]
@EvanHerman I don't see support for `.alignwide` anywhere in this PR.

**Comment by EvanHerman** [comment]
The `.alignright`, `.alignleft` classes don't appear to be rendering correctly either - which I am investigating. 

Need `blockquote` support in here as well. Still a WIP.

**Pull request merged**